sway-lib-std/src/outputs.sw

```diff
@@ -250,9 +250,9 @@ pub fn output_asset_id(index: u64) -> Option<AssetId> {
 ///     log(output_receiver);
 /// }
 /// ```
-pub fn output_asset_to(index: u64) -> Option<b256> {
+pub fn output_asset_to(index: u64) -> Option<Address> {
     match output_type(index) {
-        Output::Coin => Some(__gtf::<b256>(index, GTF_OUTPUT_COIN_TO)),
+        Output::Coin => Some(Address::from(__gtf::<Address>(index, GTF_OUTPUT_COIN_TO))),
```

Comment:
```suggestion
        Output::Coin => Some(Address::from(__gtf::<b256>(index, GTF_OUTPUT_COIN_TO))),
```
Either this or 

```suggestion
        Output::Coin => Some(__gtf::<Address>(index, GTF_OUTPUT_COIN_TO)),
```
but not sure if this one compiles

Reply:
Updated